tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+
+> The project MUST achieve a passing level badge.
+
+**Met**
+
+
+## Basic project website content
+
+> The information on how to contribute MUST include the requirements for acceptable contributions (e.g., a reference to any required coding standard). (URL required) 
+
+**Met**
+https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
+
+## Project oversight
+
+> The project SHOULD have a legal mechanism where all developers of non-trivial amounts of project software assert that they are legally authorized to make these contributions. The most common and easily-implemented approach for doing this is by using a [Developer Certificate of Origin (DCO)](https://developercertificate.org/), where users add "signed-off-by" in their commits and the project links to the DCO website. However, this MAY be implemented as a Contributor License Agreement (CLA), or other legal mechanism. (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST clearly define and document its project governance model (the way it makes decisions, including key roles). (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST adopt a code of conduct and post it in a standard location.
+
+_Possible answers: Met/Unmet_
+
+> The project MUST clearly define and publicly document the key roles in the project and their responsibilities, including any tasks those roles must perform. It MUST be clear who has which role(s), though this might not be documented in the same way. (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST be able to continue with minimal interruption if any one person dies, is incapacitated, or is otherwise unable or unwilling to continue support of the project. In particular, the project MUST be able to create and close issues, accept proposed changes, and release versions of software, within a week of confirmation of the loss of support from any one individual. This MAY be done by ensuring someone else has any necessary keys, passwords, and legal rights to continue the project. Individuals who run a FLOSS project MAY do this by providing keys in a lockbox and a will providing any needed legal rights (e.g., for DNS names).
+
+_Possible answers: Met/Unmet_
+
+> The project SHOULD have a "bus factor" of 2 or more. (URL required)
+
+_Possible answers: Met/Unmet_
+
+## Documentation
+
+> The project MUST have a documented roadmap that describes what the project intends to do and not do for at least the next year. (URL required)
+
+_Possible answers: Met/Unmet_
+
+The project MUST include documentation of the architecture (aka high-level design) of the software produced by the project. If the project does not produce software, select "not applicable" (N/A). (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST document what the user can and cannot expect in terms of security from the software produced by the project (its "security requirements"). (URL required)
+
+_Possible answers: Met/Unmet_
```

Comment:
In the [Security.md](https://github.com/nodejs/node/blob/main/SECURITY.md) we included the thread model that I believe can cover this requirement

```suggestion
> The project MUST document what the user can and cannot expect in terms of security from the software produced by the project (its "security requirements"). (URL required)

**Met**
```

Reply:
https://github.com/nodejs/node/blob/main/SECURITY.md#the-nodejs-threat-model

Reply:
Solved in 72337c0aea9317487a155de9b5cf4603ab0ffdce